src/LogExpert/Controls/LogTabWindow/LogTabWindow.cs

```diff
@@ -151,6 +154,108 @@ public LogTabWindow(string[] fileNames, int instanceNumber, bool showInstanceNum
 
         #endregion
 
+        #region ColorTheme
+        public void ChangeTheme(Control.ControlCollection container)
+        {
+            LogExpert.Config.ColorMode.LoadColorMode();
+            LogExpert.Config.ColorMode.UseImmersiveDarkMode(this.Handle, LogExpert.Config.ColorMode.DarkModeEnabled);
+
+            #region ApplyColorToAllControls
+            foreach (Control component in container)
+            {
+                if (component.Controls != null && component.Controls.Count > 0)
+                {
+                    ChangeTheme(component.Controls);
+                    component.BackColor = LogExpert.Config.ColorMode.BackgroundColor;
+                    component.ForeColor = LogExpert.Config.ColorMode.ForeColor;
+                }
+                else
+                {
+                    component.BackColor = LogExpert.Config.ColorMode.BackgroundColor;
+                    component.ForeColor = LogExpert.Config.ColorMode.ForeColor;
+                }
+
+                if (component is MenuStrip)
+                {
+                    var menu = (MenuStrip)component;
+
+                    foreach (ToolStripMenuItem item in menu.Items)
+                    {
+                        item.ForeColor = LogExpert.Config.ColorMode.ForeColor;
+                        item.BackColor = LogExpert.Config.ColorMode.BackgroundColor;
+
+                        try
+                        {
+                            for(var x = 0; x< item.DropDownItems.Count; x++)
+                            {
+                                var children = item.DropDownItems[x];
+                                children.ForeColor = LogExpert.Config.ColorMode.ForeColor;
+                                children.BackColor = LogExpert.Config.ColorMode.MenuBackgroundColor;
+
+
+                                if(children is ToolStripDropDownItem) { 
+
+                                    for (var y = 0; y < ((ToolStripDropDownItem)children).DropDownItems.Count; y++)
+                                    {
+                                        var subChildren = ((ToolStripDropDownItem)children).DropDownItems[y];
+                                        subChildren.ForeColor = LogExpert.Config.ColorMode.ForeColor;
+                                        subChildren.BackColor = LogExpert.Config.ColorMode.MenuBackgroundColor;
+                                    }
+                                }
+                            }
+                        }
+                        catch (Exception ex)
+                        {
+                            Console.WriteLine(ex.Message);

```

Comment:
if you need to log something use:

_logger.Error(e, "[errormessage]");

this should work :)

Reply:
Done!